daemon/vpn/quench/libquench.go

```diff
@@ -117,7 +130,7 @@ func (o *observer) Connecting() {
 	}
 
 	o.notifyConnectionStateChange(vpn.ConnectingState)
-	o.eventNotifier.Connected.Publish(getConnectEvent(events.StatusAttempt, o.currentServer))
+	o.eventNotifier.Connected.Publish(getConnectEvent(events.StatusAttempt, o.currentServer, o.nicName))
```

Comment:
[small] `getConnectEvent` could be a method of the `observer`, this way you won't need to pass `o.currentServer` and `o.nicName` to it as an arguments, similarly as you did in `openvpn.go`